glide.yaml

```diff
@@ -95,4 +91,4 @@ import:
 - package: github.com/pmorie/go-open-service-broker-client
   version: 31d8027f493f8f23f850415d171c7c52a972a6f2
 - package: github.com/coreos/etcd
-  version: ^3.2.9
```

Comment:
Not sure this version of etcd will let us do the embedded server trick.

Reply:
I'm thinking 3.0.10

Reply:
Interesting. I was not aware of the features we rely on in a particular version of etcd.
`3.1.10` is the version defined by `k8s.io/kubernetes`, so I just wanted it to be consistent.
If we do require `3.2.9+`, I can revert this change of course.